docs/components/modules/ROOT/pages/aws-s3-component.adoc

```diff
@@ -87,7 +87,7 @@ with the following path and query parameters:
 [width="100%",cols="2,5,^1,2",options="header"]
 |===
 | Name | Description | Default | Type
-| *amazonS3Client* (common) | Reference to a com.amazonaws.services.s3.AmazonS3 in the xref:registry.adocRegistry. |  | AmazonS3
+| *amazonS3Client* (common) | Reference to a com.amazonaws.services.s3.AmazonS3 in the xref:registry.adoc [Registry]. |  | AmazonS3
```

Comment:
This is part of the javadoc.. you need to modify the javadoc and rebuild

Reply:
Thank you Mr.Andrea, Did you mean to use mvn package command?

Reply:
when I use mvn package command every change I did revert back to where I was before.How can I fix that?

Reply:
You need to edit the javadoc related to that parameter in the S3Configuration Java class

Reply:
Okay, I found the reason from camel-website readMe. But still not clear about javadoc thing,

Reply:
If you modify the s3configuration class, you need to rebuild the module components/camel-aws-s3 with mvn clean install. After that you can go back to docs folder and run mvn clean package to have the updated adoc under ROOT

Reply:
Okay, got it, so do i have to do this for each of every component I`ve changed?

Reply:
Yes.